### PR TITLE
user guides: Review user documentation for People.

### PR DESCRIPTION
**Pull request by Privisus**
Link to GCI task: (pending, waiting for previous task)

### CODE REVIEW

File: user-guides/Privisus/people.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Reviewing user guides: People.
+
+In this document, I reviewed the following:
+
+* Check whether someone is on-line
+* Invite another user
+
+-----
+
+## Check whether someonw is on-line
```
Comment (eeshangarg) [inline user-guides/Privisus/people.md]: Replace "someonw" with "someone" :)

File: user-guides/Privisus/people.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+One final suggestion that I want to make is about redirecting people to the
+step that can't be explained so easily without retyping the entire explanation.
+I suggest add a link to **Sending messages -- Group private messages** into
+"messaged users in a group PM" (last paragraph, first sentence). This way, user
+can find out what, and how to make a group PM. If I hadn't reviewed past
+documentations, I wouldn't even know for *sure* that there exists a section for
+that.
+
+## Invita a friend to Zulip
```
Comment (eeshangarg) [inline user-guides/Privisus/people.md]: Replace `Invita` with `Invite a`. :)

File: user-guides/Privisus/people.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+to click "Invite more users" below the user list. Take a look at this figure:
+
+![Figure 1](https://lambda.sx/5Ce.png "Screenshot of Zulip's chat interface")
+
+Not only I purposefully show both "Invite users" and "Invite more users", I
+also decided it would be a good idea to just stretch the size of the figure. If
+not, I would end up having unused, weird space like this:
+
+![Figure 2](https://lambda.sx/gim.png "Unused spaces in the documentation")
+
```
Comment (eeshangarg) [inline user-guides/Privisus/people.md]: Could you maybe include both these screenshots as part of the PR, that would be very helpful! :)

File: user-guides/Privisus/people.md (hunk 1 above)
Comment (Privisus) [reply user-guides/Privisus/people.md]: Hmm, very weird. I definitely have fixed it on the last push.

File: user-guides/Privisus/people.md (hunk 2 above)
Comment (Privisus) [reply user-guides/Privisus/people.md]: This, too

File: user-guides/Privisus/people.md (hunk 3 above)
Comment (Privisus) [reply user-guides/Privisus/people.md]: Alright!

**Comment by shubhamdhama** [comment]
@Privisus Things looks fine now, but can you squash your second commit.(As only one commit is needed)

**Comment by Privisus** [comment]
Squashed commits. Should be okay now

**Comment by shubhamdhama** [comment]
Everything looks fine now.

**Pull request merged**